src/cli/cli.cpp

```diff
@@ -136,6 +136,18 @@ int Interpreter::Hex2Bin(const char *aHex, uint8_t *aBin, uint16_t aBinLength)
     return cur - aBin;
 }
 
+void Interpreter::AppendResult(ThreadError error)
+{
+    if (error == kThreadError_None)
+    {
+        sResponse.Append("Done\r\n");
+    }
+    else
+    {
+        sResponse.Append("Error\r\n");
```

Comment:
Is there any easy way to include the error code value in the printout for the error case?